Formula/civl.rb

```diff
@@ -0,0 +1,22 @@
+
+class Civl < Formula
+  desc "The Concurrency Intermediate Verification Language"
+  homepage "http://vsl.cis.udel.edu/civl/"
+  url "http://vsl.cis.udel.edu/lib/sw/civl/1.7/r3157/release/CIVL-1.7_3157.tgz"
+  sha256 "33fa5b4877fbf494b0a4b426c15d6a165fa3db48a9f97cbf391818d965d2dfb9"
+  depends_on :java => "1.8+"
+  depends_on "z3"
+  depends_on "cvc4"
+
+  def install
+    libexec.install "lib/civl-1.7_3157.jar"
+    bin.write_jar_script libexec/"civl-1.7_3157.jar", "civl"
+    share.install "doc"
+    share.install "examples"
+    share.install "licenses"
+    share.install "emacs"
+  end
+  test do
+    system "#{bin}/civl"
```

Comment:
Thank you! Please leave the pull request open. It's fine if it takes you some time to make all the adjustments and to add the test. Simply push your changes once you're done and ideally leave a comment and then I or some other maintainer will notice and get back to you.